src/internal_events/process.rs

```diff
@@ -19,13 +19,13 @@ impl InternalEvent for VectorStarted {
             build_id = built_info::VECTOR_BUILD_DESC.unwrap_or("none"),
```

Comment:
This should match the metric, imho

Reply:
changed this to "revision" with a default of empty string as well.